### ISSUE TITLE
[Accessibility] The text alternative is missing.

### ISSUE THREAD

**Issue by michalina-graczyk** [mirumee/saleor-storefront#835]
### What I'm trying to achieve is to have a text alternative.

Summary | Rule | Selector
-- | -- | --
The text alternative is missing. | alt-text-missing | ["body > div#root:nth-of-type(1) > div.home-page:nth-of-type(1) > div.products-featured:nth-of-type(2) > div.container:nth-of-type(1) > div.slider.carousel:nth-of-type(1) > div.slider-frame:nth-of-type(2) > ul.slider-list:nth-of-type(1) > li.slider-slide.slide-visible:nth-of-type(1) > a:nth-of-type(1) > div.product-list-item:nth-of-type(1) > div.product-list-item__image:nth-of-type(1) > img:nth-of-type(1)"]
The text alternative is missing. | alt-text-missing | ["body > div#root:nth-of-type(1) > div.home-page:nth-of-type(1) > div.products-featured:nth-of-type(2) > div.container:nth-of-type(1) > div.slider.carousel:nth-of-type(1) > div.slider-frame:nth-of-type(2) > ul.slider-list:nth-of-type(1) > li.slider-slide.slide-visible:nth-of-type(2) > a:nth-of-type(1) > div.product-list-item:nth-of-type(1) > div.product-list-item__image:nth-of-type(1) > img:nth-of-type(1)"]
The text alternative is missing. | alt-text-missing | ["body > div#root:nth-of-type(1) > div.home-page:nth-of-type(1) > div.products-featured:nth-of-type(2) > div.container:nth-of-type(1) > div.slider.carousel:nth-of-type(1) > div.slider-frame:nth-of-type(2) > ul.slider-list:nth-of-type(1) > li.slider-slide.slide-visible:nth-of-type(3) > a:nth-of-type(1) > div.product-list-item:nth-of-type(1) > div.product-list-item__image:nth-of-type(1) > img:nth-of-type(1)"]
The text alternative is missing. | alt-text-missing | ["body > div#root:nth-of-type(1) > div.home-page:nth-of-type(1) > div.products-featured:nth-of-type(2) > div.container:nth-of-type(1) > div.slider.carousel:nth-of-type(1) > div.slider-frame:nth-of-type(2) > ul.slider-list:nth-of-type(1) > li.slider-slide.slide-visible:nth-of-type(4) > a:nth-of-type(1) > div.product-list-item:nth-of-type(1) > div.product-list-item__image:nth-of-type(1) > img:nth-of-type(1)"]
The text alternative is missing. | alt-text-missing | ["body > div#root:nth-of-type(1) > div.home-page:nth-of-type(1) > div.products-featured:nth-of-type(2) > div.container:nth-of-type(1) > div.slider.carousel:nth-of-type(1) > div.slider-frame:nth-of-type(2) > ul.slider-list:nth-of-type(1) > li.slider-slide:nth-of-type(6) > a:nth-of-type(1) > div.product-list-item:nth-of-type(1) > div.product-list-item__image:nth-of-type(1) > img:nth-of-type(1)"]

**System information**
Operating system: MacOS
Browser: Chrome v.84
